crml/staking/src/lib.rs

```diff
@@ -1901,7 +1901,8 @@ decl_module! {
 		/// # </weight>
 		#[weight = T::WeightInfo::reap_stash(1_u32)]
 		fn reap_stash(_origin, stash: T::AccountId) {
-			ensure!(T::Currency::total_balance(&stash).is_zero(), Error::<T>::FundedTarget);
+			let at_minimum = T::Currency::total_balance(&stash) == T::Currency::minimum_balance();
+			ensure!(at_minimum, Error::<T>::FundedTarget);
```

Comment:
shouldn't this be
```suggestion
			let at_minimum = T::Currency::total_balance(&stash) <= T::Currency::minimum_balance();
			ensure!(at_minimum, Error::<T>::FundedTarget);
```

Reply:
This code is the same as Plug and possibly written according to the recent behaviour of the Balances module. Balances wouldn't allow the balance to go below the minimum. For us and with generic asset, I'm not sure it's still the case. So probably safer to check "<=".